app/src/main/res/values/strings.xml

```diff
@@ -189,4 +189,10 @@
     <!-- This is the hint value for an autofill box that appears when suggesting a value for a password field. It is interpolated with the user's account name. -->
     <string name="password_for">Password for %1$s</string>
 
+    <!-- This is used to inform the user that there are no entries in his list -->
```

Comment:
super nit: consider changing `his list` to `the list`.  There is a trend to avoid gender-specific pronouns, and I'd like to continue that trend if we can.

Reply:
👍